.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     name: Build Test
     strategy:
       matrix:
-        board: [proton_c, nice_nano, bluemicro52840_v1]
+        board: [proton_c, nice_nano, bluemicro52840_v1, nrfmicro_11, nrfmicro_11_flipped, nrfmicro_13]
```

Comment:
I think we should only merge 'nrfmicro_13' in this list, but for this branch it's nice to test all of them

Reply:
Agreed, let's leave this to `nrfmicro_13` for now.